testrunner-running/src/main/scala/edu/illinois/cs/diaper/CustomMapConverter.java

```diff
@@ -0,0 +1,47 @@
+package edu.illinois.cs.diaper;
```

Comment:
I think this package structure was re-used from some old code from PolDet, but I think we should really change it. Let's rename the package to be something like `edu.illinois.cs.statecapture`, and rename directory structure and such to match.

Reply:
Or maybe we could actually call it `edu.illinois.cs.poldet`, to give some recognition to PolDet :).